app/test/std_sdk_test.go

```diff
@@ -307,7 +307,7 @@ func (s *StandardSDKIntegrationTestSuite) TestStandardSDK() {
 			name: "signal a version change",
 			msgFunc: func() (msgs []sdk.Msg, signer string) {
 				valAccount := s.getValidatorAccount()
-				msg := signal.NewMsgSignalVersion(valAccount, 2)
+				msg := signal.NewMsgSignalVersion(valAccount, 4)
```

Comment:
```suggestion
				msg := signal.NewMsgSignalVersion(valAccount, 3)
```

Reply:
[blocking] 4 seems incorrect b/c that version isn't supported by the state machine? Why not use 3 here like Nina suggests.

Reply:
Because 3 is the current version. It would be weird to signal for the current version. What is more normal is to signal for the next version

Reply:
I could do something like `LatestVersion + 1` so we don't have to continually increment it

Reply:
Oh I see, thanks! No longer blocking.

4 or `LatestVersion + 1` both work for me.